src/database/sparql_implementation.py

```diff
@@ -128,8 +140,7 @@ def get_sssom_mappings_query(self, filter: Union[List[dict], None]) -> Iterable[
     default_query = self.add_filters(self.default_query(Mapping.class_class_uri, self.schema_view.mapping_slots.copy()), filter)
     bindings = self._query(default_query)
     for row in bindings:
-      r = self.transform_result(row)
-      r.pop("_x")
+      r = self.add_id(self.transform_result(row), "sssom_api:mapping_id")
```

Comment:
Always use static variables instead of strings in code. We typically recommend a specific file like "constants.py" in the project setup that contains all the static variables:

```
SSSOM_API_MAPPING_ID = "sssom_api:mapping_id"
```

Reply:
done